highfidelity/hifi_tests
# 20286 - fix protocol tests

https://highfidelity.fogbugz.com/f/cases/20286/

# Description
1.  The PolyVox protocol test is wrong.  The test was failing, but nitpick didn't catch the error due to a bug in the test validation code.

2.  The result of the `render / engine / camera / primary` test depends on the size of the avatar.  Modify to use fixed avatar
# Testing
1.  Stop any running instances  of Sandbox and Interface.
2.  Create a folder for test results (e.g. - D:\tests)
3.  On Windows - open a terminal in the High Fidelity program folder.
4.  Run the sandbox 
5.  Run the following commands:
`start domain-server`
`start assignment-client -n 6`
`interface.exe --url hifi://localhost --no-updater --no-login-suggestion --testScript https://raw.githubusercontent.com/NissimHadar/hifi_tests/20286-fixProtocolTests/tests/protocol/polyvox/testAuto.js quitWhenFinished --testResultsLocation D:/temp`.  
Replace D:/temp with the test results folder created in step 1
6.  After completion, verify that a file named `tests.protocol.polyvox.00000.passed.txt` exists in the test results folder
7.  Delete `tests.protocol.polyvox.00000.passed.txt`
8.  Run the following command:
`interface.exe --url hifi://localhost --no-updater --no-login-suggestion --testScript https://raw.githubusercontent.com/NissimHadar/hifi_tests/20286-fixProtocolTests/tests/engine/render/camera/primary/testAuto.js quitWhenFinished --testResultsLocation D:/temp`.  
9.  Use nitpick v1.2 to evaluate the results.
